src/components/header/header/header.ts

```diff
@@ -158,6 +163,9 @@ export class SbbHeaderElement extends LitElement {
   }
 
   private _closeOpenOverlays(): void {
+    if (this.hasAttribute('data-has-visible-focus-within')) {
```

Comment:
I think this should be part of the new controller, so we don't have to expose the name of the data attribute

Reply:
I'll try to implement this improvement in the dialog header PR since I need the same logic there too 👍

Reply:
perfect, thanks!